rocAL/rocAL/source/meta_data/meta_node_resize_mirror_normalize.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "meta_node_resize_mirror_normalize.h"
+
+void ResizeMirrorNormalizeMetaNode::initialize()
+{
+    _mirror_val.resize(_batch_size);
+}
+void ResizeMirrorNormalizeMetaNode::update_parameters(pMetaDataBatch input_meta_data, pMetaDataBatch output_meta_data)
+{
+    initialize();
+    if(_batch_size != input_meta_data->size())
+    {
+        _batch_size = input_meta_data->size();
+    }
+    _mirror = _node->return_mirror();
```

Comment:
return_mirror - Is this function name correct?
We can ask get_mirror. return_mirror does not seem right to me please check

Reply:
The function name is adapted from node CMN. I'll make the change for node RMN